packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -256,21 +256,31 @@ export const AskSubscriptionQuestion: FuncQuestion = {
   },
 };
 
-export const GetUserEmailQuestion: TextInputQuestion = {
-  name: "email",
-  type: "text",
-  title: "Add owner to Teams/AAD app for the account under the same M365 tenant (email)",
-  validation: {
-    validFunc: (input: string, previousInputs?: Inputs): string | undefined => {
-      if (!input || input.trim() === "") {
-        return "Email address cannot be null or empty";
-      }
+export function getUserEmailQuestion(currentUserEmail: string): TextInputQuestion {
+  const defaultUserEmail = "[UserName]@" + currentUserEmail.split("@")[1];
```

Comment:
Do we need check whether currentUserEmail is null?

Reply:
Updated